scripts/eosio_build_darwin_deps

```diff
@@ -4,9 +4,9 @@ Libtool,-x,/usr/local/bin/glibtool,libtool,http://gnu.askapache.com/libtool/libt
 OpenSSL,-f,/usr/local/opt/openssl/lib/libssl.a,openssl,https://www.openssl.org/source/openssl-1.0.2n.tar.gz
 wget,-x,/usr/local/bin/wget,wget,https://ftp.gnu.org/gnu/wget/wget-1.19.2.tar.gz
 GMP,-f,/usr/local/opt/gmp/include/gmpxx.h,gmp,https://ftp.gnu.org/gnu/gmp/gmp-6.1.2.tar.bz2
-llvm,-x,/usr/local/opt/llvm@4/bin/clang-4.0,llvm@4,http://releases.llvm.org/4.0.1/llvm-4.0.1.src.tar.xz
-python,-d,/usr/local/Cellar/python/3.7.2_1,python,https://www.python.org/ftp/python/3.7.2/Python-3.7.2.tgz
-python@2,-d,/usr/local/Cellar/python@2/2.7.15_2,python@2,https://www.python.org/ftp/python/2.7.13/Python-2.7.13.tgz
+llvm,-d,/usr/local/opt/llvm@4,llvm@4,http://releases.llvm.org/4.0.1/llvm-4.0.1.src.tar.xz
+python,-d,/usr/local/Cellar/python/3\*,python,https://www.python.org/ftp/python/3.7.2/Python-3.7.2.tgz
```

Comment:
Looking in the Cellar is a bad idea IMO -- it's an implementation detail of sorts. Can't we look at `/usr/local/opt/python@2/` and `/usr/local/opt/python` as a more appropriate place?

Reply:
I was thinking about that too, it would simplify things up until the point python 4 is released and uses `/usr/local/opt/python`. Do we want to accommodate for that future situation?

Reply:
That may not end up being a problem: is python 4 going to be a breaking change like 2->3 was?

Reply:
I think you're right actually. Probably not a big deal for now